scripts/install_third_party_libs_test.py

```diff
@@ -89,6 +95,31 @@ def mock_ensure_directory_exists(unused_path):
         self.dir_exists_swap = self.swap(
             common, 'ensure_directory_exists', mock_ensure_directory_exists)
 
+    def test_install_prerequisite(self) -> None:
+        prerequisite = install_third_party_libs.PREREQUISITES[0]
+        with self.Popen_swap:
+            install_third_party_libs.install_prerequisite(prerequisite)
+        self.assertTrue(self.check_function_calls['check_call_is_called'])
+
+    def test_install_prerequisite_with_prefix_fix(self) -> None:
+        self.check_function_calls['check_call_is_called'] = False
+        prerequisite = install_third_party_libs.PREREQUISITES[0]
+        with self.Popen_error_swap:
+            with self.check_call_swap:
+                install_third_party_libs.install_prerequisite(prerequisite)
+        self.assertTrue(self.check_function_calls['check_call_is_called'])
+
+    def test_install_prerequisite_raises_exception_if_prefix_fix_fails(
+        self) -> None:
+        self.check_function_calls['check_call_is_called'] = False
+        prerequisite = install_third_party_libs.PREREQUISITES[0]
+        with self.Popen_error_swap:
+            with self.check_call_error_swap:
+                with self.assertRaisesRegex(
+                    Exception, 'Error installing prerequisite'):
```

Comment:
ditto: indent one more level

Reply:
> ditto: indent one more level

Little bit confused about this: it doesn't seem to be a standard followed by the rest of the file. See lines 313, 329. Should I change those as well?

Reply:
Regardless, resolved.

Reply:
Yes, we used to have a lint check that didn't accept indentation by two levels in situations like this. You don't need to change the other instances, but you're welcome to. I find single-level indentations pretty unreadable in situations like this